apps/docs/src/.vitepress/components/OnyxIconLibrary.vue

```diff
@@ -48,7 +48,9 @@ const filteredCategories = computed(() => {
       <OnyxHeadline is="h3" class="category__headline">{{ category.name }}</OnyxHeadline>
 
       <div class="category__icons">
-        <IconLibraryItem v-for="icon in category.icons" :key="icon.iconName" :icon="icon" />
+        <ClientOnly>
```

Comment:
Please add a commen to explain why this is necessary

Reply:
Because the build failed because the `createTooltip` composable which is used in the OnyxTooltip called `document.addEventListener()` directly but the document is not available in SSR.

I fixed it to only add the listeners onBeforeMount